requirements.txt

```diff
@@ -0,0 +1,4 @@
+bpy==3.6.0
```

Comment:
Can you combine the 2 `requirements.txt` together? It would also be nice to note somewhere that we need to use python 3.10 for this project

Reply:
Yea, this issue should cover explaining the set up
https://github.com/Lenscrafters-Capstone/LensCrafters-free/issues/18